### ISSUE TITLE
[enhancement] Warp Coils

### ISSUE THREAD

**Issue by gdarai** [zmaster587/AdvancedRocketry#935]
Hello ... I have an enhancement which would make the warpship even more StarTrek-like.

Did you ever thought about Warp Coils?
Warp Coil is an object... usually hiding in your nacelles (The long things at the back of a star trek ship)
![engine_nacell](https://user-images.githubusercontent.com/26735549/32634127-6d3815f0-c5aa-11e7-8c07-0ce879defe24.jpg)
![scr02](https://user-images.githubusercontent.com/26735549/32634139-78d318f6-c5aa-11e7-81d8-4dd4c9dd30ce.png)

The faster ship you ant the more coils it needs.

Well in AdvancedRocketry it could be like this:
You need certain number of Coils on your ship, all facing the coorect direction to be able to travel certain distance. And all has to be powered.
The number of Coils per distance unit or consumed dilithium can be configurable and so can be the required power input into one nacelle.

What do you think?

**Comment by dmodoomsirius** [comment]
Taking enhancement out of the title as i gave it the tag.

**Comment by C1ffisme** [comment]
The ship shouldn't be faster for having more coils, but it should be able to warp further.

I kinda like the idea of a customizable multiblock like this, with a balance between efficiency and travel distance. (Not travel time, though. I don't like waiting for long periods of time with nothing interesting to do in-between.)

**Comment by gdarai** [comment]
I agree... it should influence definitely distance, not time.

Other idea is to make the diffrence between distance and number of coils non-linear... so it would not be such a greate difference between travel to next planet and travel to next star. But right configuration parameters could easily make this work.

Example of distances: If you have distance between stars 100 and differene between planets also 100, let's expect alpha centauri (5 LY = 2.5MLmin) and the second earth (10Lmin)... So let's say we can set that with every coil your ship can travel a distance with next power of 10 of light minutes. So...
1 coil = 1Lmin = 10 distance in-system
2 coils = 10Lmin (sun-earth) = 100 distance in-system
3 coils = 100 Lmin (entire solar system) = 1000 distance in-system
4 coils = 1'000 Lmin = 0.6 Lday
5 coils = 10'000 Lmin = 6 Lday = maybe a very close-by star (0.6 on the galaxy map)
6 coils = 100'000 Lmin = 2Lmonths = 6 on the galaxy map
7 coils = 1'000'000 Lmin = 20Lmonths = 60 on the galaxy map, a decent range.

The math:
So I see here a working example, where the coils create a distance A*(B^(count)) in the case above A = 1, B = 10. This should make it configurable enough even for the hardcore map creators who want to make their player's lives hell.

**Comment by CTH999** [comment]
I agree. I also came up with an idea for an alternative fuel system [Alternative Fuels](https://github.com/zmaster587/AdvancedRocketry/issues/945l) Maybe it could include other types of matter on planets. (Leaving comment there with more info)

**Comment by NutMaster6942** [comment]
Higher tier coils should cut down on fuel cost.
